charts/aws-mountpoint-s3-csi-driver/templates/mountpoints3podattachments-crd.yaml

```diff
@@ -17,7 +17,23 @@ spec:
     singular: mountpoints3podattachment
   scope: Cluster
   versions:
-  - name: v2beta
+  - additionalPrinterColumns:
+    - description: The node where the volume is mounted
+      jsonPath: .spec.nodeName
+      name: Node
+      type: string
+    - description: The persistent volume name
+      jsonPath: .spec.persistentVolumeName
+      name: PV Name
+      type: string
+    - description: Comma separated mount options
+      jsonPath: .spec.mountOptions
+      name: Mount Options
+      type: string
+    - jsonPath: .metadata.creationTimestamp
+      name: Age
+      type: date
```

Comment:
Should we call it creation timestamp instead? Age seems inaccurate given the value being a date

Reply:
Most resources use `AGE` for this field, wanted to be consistent with them

```bash
$ kubectl get po busybox
NAME      READY   STATUS    RESTARTS   AGE
busybox   1/1     Running   0          47h
$ kubectl get pv s3-mp-csi-pv
NAME           CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                   STORAGECLASS   VOLUMEATTRIBUTESCLASS   REASON   AGE
s3-mp-csi-pv   1Gi        RWX            Retain           Bound    default/s3-mp-csi-pvc                  <unset>                          47h